### DISCUSSION TITLE
Allow array to be passed as a workflow dispatch event input

### DISCUSSION THREAD

**Discussion by christos97** [community/community#15877] [Actions]
Hello GitHub team!

### This is a feature request.

It would be really nice to be able to "natively" pass an array as an input to a workflow dispatch event.

I know these values get loaded into env vars, hence the key-value pairs only restriction.
It is possible to work-around using a stringified array and parse it inside the workflow, but it would be very nice to be able to 
declare an array when manually triggering a workflow, even select your array/batch from the dropdown list of the actions gui.

### This would be very helpful for versioning a SaaS product in client batches and easily spawn jobs by using the matrix strategy down the road. 

Thank you very much

**Comment by Bigjohn88** [comment]
I like
